src/views/CreateProposal/index.tsx

```diff
@@ -380,8 +433,8 @@ function CreateProposal() {
                 variant="body1"
                 sx={{ width: desktop ? "220px" : "100%", mt: 3 }}
               >
-                Creating a Proposal costs {proposalcost}. Please make sure everything
-                is correct.
+                Creating a Proposal costs {proposalcost}. Please make sure
```

Comment:
When a user confirms and submits a proposal the page should reload to the details page of that proposal

Reply:
It's not possible with the current implementation of the contract. I will redirect to all Crowdfundings.